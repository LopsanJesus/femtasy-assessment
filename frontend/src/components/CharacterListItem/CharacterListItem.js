import React from 'react';
import './CharacterListItem.scss';

const CharacterListItem = () => {
  return <div>CharacterListItem</div>;
};

export default CharacterListItem;
