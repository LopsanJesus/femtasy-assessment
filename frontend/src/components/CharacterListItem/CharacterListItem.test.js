import React from 'react';
import { shallow } from 'enzyme';
import CharacterListItem from './CharacterListItem';

describe('CharacterListItem', () => {
  test('matches snapshot', () => {
    const wrapper = shallow(<CharacterListItem />);
    expect(wrapper).toMatchSnapshot();
  });
});
