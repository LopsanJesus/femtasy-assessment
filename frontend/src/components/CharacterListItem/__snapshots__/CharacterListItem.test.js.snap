// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterListItem matches snapshot 1`] = `ShallowWrapper {}`;
